ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.model.js

```diff
@@ -28,13 +29,20 @@ module.exports = Backbone.AssociatedModel.extend({
         mouseLon: undefined,
         clickLat: undefined,
         clickLon: undefined,
+        dms: undefined,
         target: undefined,
         targetMetacard: undefined
     },
     updateClickCoordinates: function () {
+
```

Comment:
Nit: Remove this space. Other than that LG2M 👍

Reply:
Removed